Chapter-3-microservice-extraction/Fitnet/Src/Passes/Tests/Fitnet.Passes.IntegrationTests/RegisterPass/RegisterPassTests.cs

```diff
@@ -33,12 +33,5 @@ internal async Task Given_contract_signed_event_Then_should_register_pass()
 
         // Assert
         _testEventBus.EnsureConsumed<ContractSignedEvent>();
-        await EnsureThatPassRegistered();
-    }
-
-    private async Task EnsureThatPassRegistered()
```

Comment:
https://github.com/evolutionary-architecture/evolutionary-architecture-by-example/issues/45
Issue Created to restore this change